test/expand.coffee

```diff
@@ -30,21 +32,21 @@ testExpand(
 # String object
 testExpand(
 	a: 'b'
-	'a/b'
+	new Error(''''$expand: a: "b"' is invalid, use '$expand: a: $expand: "b"' instead.''')
```

Comment:
Is `''''` coffeescript syntax? I haven't seen it before

Reply:
`'''` is coffeescript syntax (so you don't need to quote `'` or `"`), the extra `'` is the start of the string